### PR TITLE
couple bug fixes

### PR DESCRIPTION
**Pull request by xualber**
Changed the bucket definition to default bucket. Works on both NI and Studio now.

### CODE REVIEW

**Comment by xualber** [comment]
Made another change.

File: introduction_to_amazon_algorithms/factorization_machines_mnist/factorization_machines_mnist.ipynb [hunk 1]
```diff
@@ -64,15 +64,21 @@
...
     "# Define IAM role\n",
-    "import boto3\n",
     "import re\n",
+    "import boto3\n",
+    "import sagemaker\n",
     "from sagemaker import get_execution_role\n",
+    "import json\n",
+    "role = get_execution_role()\n",
+    "\n",
+    "role = get_execution_role()\n",
```
Comment (aaronmarkham) [inline introduction_to_amazon_algorithms/factorization_machines_mnist/factorization_machines_mnist.ipynb:75]:
```suggestion

```
Duplicated line.

Comment (aaronmarkham) [inline introduction_to_amazon_algorithms/factorization_machines_mnist/factorization_machines_mnist.ipynb:75]: Still seeing double here....

**Comment by aaronmarkham** [comment]
Please rebase this @xualber

**Pull request closed without merging**